lib/db.js

```diff
@@ -418,20 +455,37 @@ module.exports = (
   }
 
   DB.prototype.updateSessionToken = function (token, userAgentString) {
```

Comment:
I hadn't thought about the delete part before, nice one for catching it @vbudhram and sorry for not mentioning it before @udara!

If we want parity with the db, and I think we do, it means session tokens should be removed from redis in each of the following:

* `deleteSessionToken`
* `deleteAccount`
* `resetAccount`

I think that's it, unless there are others I've forgotten about?